sklearn/feature_extraction/tests/test_dict_vectorizer.py

```diff
@@ -209,3 +209,20 @@ def test_dictvectorizer_dense_sparse_equivalence():
     expected_inverse = [{"category=thriller": 1.0}]
     assert dense_inverse_transform == expected_inverse
     assert sparse_inverse_transform == expected_inverse
+
+
+def test_dict_vectorizer_unsupported_value_type():
+    """Check that we raise an error when the value associated to a feature
+    is not supported.
+    Non-regression test for:
+    https://github.com/scikit-learn/scikit-learn/issues/19489
+    """
```

Comment:
Small nitpick for [PEP 257 multi-line docstrings](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings).
```suggestion
    """Check that we raise an error when the value associated to a feature
    is not supported.

    Non-regression test for:
    https://github.com/scikit-learn/scikit-learn/issues/19489
    """
```